src/main/webapp/crearEncuesta.jsp

```diff
@@ -1,14 +1,64 @@
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <!DOCTYPE html>
-<html lang="en">
+<html lang="es">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Crear Encuesta</title>
-    <link rel="stylesheet" href="styles.css"> <!-- Incluye un CSS opcional para diseño -->
+    <title>Crear Encuesta y Pregunta</title>
+    <!-- Enlace a Bootstrap CDN -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <style>
```

Comment:
Si tenéis una hoja de estilos ya enlazada moved estos estilos a esa hoja, no es una buena práctica usar ya la etiqueta styles